lib/Transforms/IPO/PassManagerBuilder.cpp

```diff
@@ -504,6 +504,17 @@ void PassManagerBuilder::populateModulePassManager(
   }
 
   // HLSL Change Begins.
+  {
+    // Run reassociate pass again after GVN since GVN will expose more
+    // opportunities for reassociation.
+    if (HLSLEnableAggressiveReassociation) {
+      MPM.add(createReassociatePass(true)); // Reassociate expressions
+      if (EnableGVN) {
+        MPM.add(createGVNPass(DisableGVNLoadPRE)); // Remove redundancies
+      }
```

Comment:
nit:
```suggestion
      if (EnableGVN)
        MPM.add(createGVNPass(DisableGVNLoadPRE)); // Remove redundancies
```

See: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
Done.